Core/Core/Discussions/DiscussionReplyViewController.swift

```diff
@@ -176,7 +176,8 @@ public class DiscussionReplyViewController: ScreenViewTrackableViewController, E
     }
 
     func heightChanged() {
-        let themeSwitchButtonOffset: CGFloat = traitCollection.userInterfaceStyle == .dark ? 38 : 0
+        let themeSwitcherHeight = webView.themeSwitchButtonHeight + webView.themeSwitchButtonTopPadding
```

Comment:
Since there's no external factor, this value could originate from the webView already.

Reply:
Good idea.